libraries/postgres/@types/fns.d.ts

```diff
@@ -0,0 +1,3393 @@
+// Generated type definitions for DB functions
```

Comment:
If this is a third party library there may be a `@types` module available. If this is generated locally to the repo it would be nice to include instructions in the header how to regenerate it.